master/internal/api_project.go

```diff
@@ -1076,3 +1076,31 @@ func (a *apiServer) GetProjectsByUserActivity(
 
 	return &apiv1.GetProjectsByUserActivityResponse{Projects: viewableProjects}, nil
 }
+
+func (a *apiServer) GetMetadataValues(
+	ctx context.Context, req *apiv1.GetMetadataValuesRequest,
+) (*apiv1.GetMetadataValuesResponse, error) {
+	curUser, _, err := grpcutil.GetUser(ctx)
+	if err != nil {
+		return nil, err
+	}
+	p, err := a.GetProjectByID(ctx, req.ProjectId, *curUser)
+	if err != nil { // Can view project?
+		return nil, err
+	}
+
+	values := []string{}
+	err = db.Bun().NewSelect().Distinct().
```

Comment:
style nit: Could you add comments letting folks know that this is only retrieving string typed metadata? It might be confusing if someone thinks this is also supposed to return all the other metadata key-value pairs

Reply:
It also might be worth moving the db logic over into the `postgres_project.go`

Reply:
What is the significance of adding Distinct to this query?

Reply:
Distinct() ensures that we are getting unique values.